libraries/resource_providers/api200/server_profile_provider.rb

```diff
@@ -30,6 +30,29 @@ def set_connections(type, connection_list)
         true
       end
 
+      def build_volume_attachments
+        return unless @new_resource.volume_attachments
+        @new_resource.volume_attachments.each do |options|
+          options = convert_keys(options, :to_s)
+          volume_name = options['volume']
+          volume_data = options['volume_data']
+          attachment_data = options.fetch('attachment_data', {})
+          raise('To add volume attachments you need volume or volume_data into each data of volume_attachments') unless volume_name || volume_data
```

Comment:
Probably rewrite message.

Reply:
Yeah, maybe `"To add volume attachments you need inform volume or volume_data"` is good.

Reply:
A 'volume' or 'volume_data' is required for each volume attachment specified.